kroxylicious-kms-provider-hashicorp-vault-test-support/src/main/java/io/kroxylicious/kms/provider/hashicorp/vault/model/CreatePolicyRequest.java

```diff
@@ -4,17 +4,17 @@
  * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
  */
 
-package io.kroxylicious.kms.provider.hashicorp.vault;
+package io.kroxylicious.kms.provider.hashicorp.vault.model;
 
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.UncheckedIOException;
 import java.nio.charset.StandardCharsets;
 import java.util.Objects;
 
-record CreatePolicyRequest(String policy) {
+public record CreatePolicyRequest(String policy) {
```

Comment:
I'm not convinced we actually need this change (that assumes I'm right about the Test Kek Managers)

Reply:
I moved all the `*Request*` classes to a `model` package to have the same structure than the other KMS (AWS is structured using the model package and public records)

Reply:
We really should adopt Project Jigsaw so we'd have properly modularity.    :)